CefSharp.Wpf.Example/Views/BrowserTabView.xaml.cs

```diff
@@ -27,7 +39,12 @@ public BrowserTabView()
 
             browser.RequestHandler = new RequestHandler();
             browser.RegisterJsObject("bound", new BoundObject(), BindingOptions.DefaultBinder);
-            browser.RegisterAsyncJsObject("boundAsync", new AsyncBoundObject());
+            var bindingOptions = new BindingOptions() 
```

Comment:
Add a comment here as to what feature your demonstrating and it's purpose please.